tests/kuttl/test-ephemeral-gateway/test_creds.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 
 export EPHEM_BASE64=`kubectl get secret test-ephemeral-gateway-keycloak -n test-ephemeral-gateway -o json | jq '.data | map_values(@base64d)' | jq -r -j '"\(.defaultUsername):\(.defaultPassword)" | @base64'`
-export PODNAME=`kubectl get pod -l env-app=test-ephemeral-gateway-keycloak -n test-ephemeral-gateway -o json | jq -r '.items[0].metadata.name'`
```

Comment:
Because before we were running it inkeycloak which was _ok_ but not the best.....now keycloak dropped support for curl, so.......they forced me to do a better thing